From 69e3cbb5ce27693819ede28d673cd6a980933996 Mon Sep 17 00:00:00 2001
From: umeshuday <umeshuday788@gmail.com>
Date: Sat, 28 Jan 2023 15:24:10 +0530
Subject: [PATCH] intial umesh commit

---
 file.js | 16 +++++++++++++++-
 1 file changed, 15 insertions(+), 1 deletion(-)

diff --git a/file.js b/file.js
index 9ab9ebd..b4b6c28 100644
--- a/file.js
+++ b/file.js
@@ -1,3 +1,17 @@
 const commmand ="1"
 
-const bla = " blaa blaa"
\ No newline at end of file
+const bla = " blaa blaa"
+
+export const usePagination = ({
+    totalCount,
+    pageSize,
+    siblingCount = 1,
+    currentPage
+  }) => {
+    const paginationRange = useMemo(() => {
+       // Our implementation logic will go here 
+        
+    }, [totalCount, pageSize, siblingCount, currentPage]);
+  
+    return paginationRange;
+  };
\ No newline at end of file
-- 
2.39.0.windows.2

